pageserver/src/page_service.rs

```diff
@@ -723,7 +757,7 @@ impl PageServerHandler {
         // Import wal provided via CopyData
         info!("importing wal");
         pgb.write_message_noflush(&BeMessage::CopyInResponse)?;
-        self.flush_cancellable(pgb, &timeline.cancel).await?;
+        self.flush_cancellable(pgb, None).await?;
         let mut copyin_reader = pin!(StreamReader::new(self.copyin_stream(pgb, &timeline.cancel)));
```

Comment:
Why does the flush_cancellable() call here not take Some(&timeline.cancel) ?

Reply:
This was a mistake: I was writing this call thinking that the timeline was fetched via cache and so would be implicitly covered by flush_cancellable.  Fixed in 013a8ab1f18603e1f29add3a6a84c0706e46e02e